example_dags/example_amazon_s3_snowflake_transform.py

```diff
@@ -46,19 +46,33 @@ def aggregate_data(df: pd.DataFrame):
     catchup=False,
 )
 def example_amazon_s3_snowflake_transform():
+    input_table_1 = Table(
+        "ADOPTION_CENTER_1",
+        database=os.environ["SNOWFLAKE_DATABASE"],
+        schema=os.environ["SNOWFLAKE_SCHEMA"],
+        conn_id="snowflake_conn",
+    )
+    input_table_2 = Table(
+        "ADOPTION_CENTER_2",
+        database=os.environ["SNOWFLAKE_DATABASE"],
+        schema=os.environ["SNOWFLAKE_SCHEMA"],
+        conn_id="snowflake_conn",
+    )
+
+    temp_table_1 = aql.load_file(
+        path="s3://tmp9/ADOPTION_CENTER_1.csv",
+        file_conn_id="",
+        output_table=input_table_1,
+    )
+    temp_table_2 = aql.load_file(
+        path="s3://tmp9/ADOPTION_CENTER_2.csv",
```

Comment:
I see some inconsistencies with using env vars for certain fields and not for others.

We could do same as OSS so a dag can be run. For example:

```python
        s3_bucket = os.get_environ.get("S3_BUCKET", "s3://tmp9")
        path=f"{s3_bucket}/ADOPTION_CENTER_1.csv",
```

Reply:
This way users can too run them by just setting up environment variables without having to change the Example DAG, wdyt?

Reply:
This does not need to block this PR but can you create a backlog issue to make example DAGs consistent with that pattern if you agree

Reply:
@kaxil Yes agreed I'll make this change for the current dag, and I'll create a ticket for the remaining dags.